spec/observables/from-callback-spec.js

```diff
@@ -0,0 +1,93 @@
+/* globals describe, it, expect */
+var Rx = require('../../dist/cjs/Rx');
+var Observable = Rx.Observable;
+
+describe('Observable.fromCallback', function () {
+  it('should emit one value from a callback', function (done) {
+    function callback (datum, cb) {
+      cb(datum);
+    }
+    var cbToObs = Observable.fromCallback(callback);
+
+    cbToObs(42)
+      .subscribe(function (x) {
+        expect(x).toBe(42);
+      }, null,
+      function (x) {
+        expect(x).toBe(undefined);
```

Comment:
Since this is the `complete()` Observer function, it never expects any parameter, so you can remove `function(x)`. Otherwise the test case might be read in a awkward way, **as if** usually `complete()` takes a parameter. Actually you can altogether replace this function with just `done`.

Also, it is customary to use `done.fail` as the `error()` handler. So replacing `null` with `done.fail` is a good idea in order to catch unexpected error that a test case could possibly raise.